pkg/controller/schedule_controller.go

```diff
@@ -89,6 +89,11 @@ func (c *scheduleReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 		return ctrl.Result{}, errors.Wrapf(err, "error getting schedule %s", req.String())
 	}
 
+	if schedule.Spec.Paused {
```

Comment:
There is already status.Phase `SchedulePhaseEnabled` providing similar functionality.
I know the purpose of the phase is used to indicate schedule is validated, but it's still a little confusing, and I think schedule controller's status checking logic needs modification. Maybe some refactoring is also needed.